frontend/src/hooks/useClientFetch.ts

```diff
@@ -12,7 +12,7 @@ import { useCallback } from "react";
     * throwing errors on unsuccessful requests
  */
 export const useClientFetch = <T>(
-  errorMessage: string,
+  eMessage: string,
```

Comment:
this isn't necessary - if error is part of the name of another variable we can leave it be